tests/test_utils.py

```diff
@@ -130,3 +132,41 @@ def test_get_occurrence_works_for_event_like_object(self):
             expected_ok = True
 
         assert expected_ok
+
+
+class BaseTestClass(object):
+    pass
```

Comment:
Agree, it's ugly, but how to test the `import_string`. What I've written here is not really a unit test.
I should have use `mock` lib to mock that `import_string`. Should I add `mock`? I see you're not using it.